pkg/operator/starter.go

```diff
@@ -105,10 +108,40 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	)
 
+	staticResourceController := staticresourcecontroller.NewStaticResourceController(
```

Comment:
future me is strongly in favor a `directory/...` mechanism.

Reply:
yes - there will be a _lot_ of controller RBAC moving over in this refactor.